daft/daft.pyi

```diff
@@ -1546,6 +1546,7 @@ class PhysicalPlanScheduler:
     ) -> PhysicalPlanScheduler: ...
     def num_partitions(self) -> int: ...
     def repr_ascii(self, simple: bool) -> str: ...
+    def display_as(self, options: dict[str, Any]) -> str: ...
```

Comment:
preference would be to have the options listed out here

Reply:
yeah I was struggling on getting solid typing working for this, I'll take another attempt and see if i can come up with something